### ISSUE TITLE
Suggestion and discussion: nice pad urls without the "/p" part (service urls remain unchanged)

### ISSUE THREAD

**Issue by Wikinaut** [ether/etherpad-lite#367]
I suggest to slightly modify the server.js module, so that urls of the nice form /padname (where padname is different from the few other service urls as used by etherpad-lite, see list below) can be used as an alternative to the canonical form /p/padname which will remain working.

When having the proposed method applied, padnames which begin with one of the following strings are not allowed any more (they are still allowed in the canonical form "/p/padname"):
/static/
/socket.io/
/ep/
/minified/
/api/
/ro/
/error/
/jserror
/favicon.ico
/robots.txt

Basically see my module https://github.com/Wikinaut/etherpad-lite/blob/master/node/server.js for a working version, forked some time ago.

If users want to use such padnames, they can still use the canonical form, e.g. /p/favicon.ico to work with such a pad as before.

Please let me know if you want me to post a corresponding patch, and a pull request. I am looking forward to your answers.
